Library/Models/SurveyDataRequest.cs

```diff
@@ -91,10 +91,25 @@ public class SurveyDataRequest
         public bool IncludeParaData { get; set; }
 
         /// <summary>
-        /// Include in the download the captured media files during the interviews
-        /// Previously: DownloadCapturedMedia
+        /// Include in the download the All captured Media Files during the interviews (AudioQuestion, VideoQuestion, PhotoQuestion, AudioSilentRecording)
         /// </summary>
         public bool IncludeCapturedMediaFiles { get; set; }
+        /// <summary>
+        /// Include Audio Silent Recording captured Files during the interviews. This parameter will be ignored if IncludeCapturedMediaFiles is true
+        /// </summary>
+        public bool IncludeCapturedAudioSilentRecordingFiles { get; set; } = false;
```

Comment:
Is there a difference with added setting = false instead of omitting it? As false is the default for a boolean.

Reply:
It was to remark that is false, but no there are no differences.